errors/errors.go

```diff
@@ -36,4 +36,5 @@ var (
 	ErrIllegalConfigKey        = errors.New("cli: given config key is not allowed")
 	ErrScanNotSupported        = errors.New("search: scanning of image media type not supported")
 	ErrFixedTagNotFound        = errors.New("search: no fixed tag found")
+	ErrCLITimeout              = errors.New("cli: Query timed out waiting for results")
```

Comment:
I think it should be errors.New("cli: Query timed out while waiting for results")

Reply:
yeah, sounds better 👍